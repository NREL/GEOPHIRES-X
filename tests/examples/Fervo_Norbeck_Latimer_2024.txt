--- Case Study: Fervo Energy Whitepaper
--- Norbeck, J.H. and Latimer, T.M. 2024.
---   "Commercial-Scale Demonstration of a First-of-a-Kind Enhanced Geothermal System."
---   https://doi.org/10.31223/X52X0B

Reservoir Model, 1
Reservoir Depth, 2347 meter
Number of Segments, 1
Gradient 1, 68
Number of Production Wells, 1
Number of Injection Wells, 1
Production Well Diameter, 7
Injection Well Diameter, 7
Ramey Production Wellbore Model, 1
Production Wellbore Temperature Drop, .5
Injection Wellbore Temperature Gain, 0
Production Flow Rate per Well, 63
Fracture Shape, 3
Fracture Height, 900
Reservoir Volume Option, 3
Reservoir Volume, 1000000000
Number of Fractures, 20
Water Loss Fraction, .02
Productivity Index, 5
Injectivity Index, 5
Injection Temperature, 50
Maximum Drawdown, 1
Reservoir Heat Capacity, 1000
Reservoir Density, 2700
Reservoir Thermal Conductivity, 2.7
End-Use Option, 1
Economic Model, 1
Power Plant Type, 3
Circulation Pump Efficiency, .8
Utilization Factor, .9
Surface Temperature, 20
Ambient Temperature, 20
Plant Lifetime, 30
Fixed Charge Rate, .05
Inflation Rate During Construction, 0
Time steps per year, 6
Maximum Temperature, 500
Well Drilling and Completion Capital Cost, 40
