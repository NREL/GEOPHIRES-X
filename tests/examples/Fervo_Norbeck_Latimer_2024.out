                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.28
 GEOPHIRES Build Date: 2024-03-05
 Simulation Date: 2024-05-06
 Simulation Time:  09:14
 Calculation Time:      0.675 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     2.45 MW
      Electricity breakeven price:                           40.19 cents/kWh
      Number of production wells:                             1
      Number of injection wells:                              1
      Flowrate per production well:                          63.0 kg/sec
      Well depth (or total length, if not vertical):          2.3 kilometer
      Geothermal gradient:                                    0.0680 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model = Fixed Charge Rate (FCR)
      Fixed Charge Rate (FCR):                                5.00
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                        -131.75 MUSD
      Project IRR:                                           0.00 %
      Project VIR=PI=PIR:                                   -0.03
      Project MOIC:                                         -0.81
      Project Payback Period:                          N/A

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             1
      Number of Injection Wells:                              1
      Well depth (or total length, if not vertical):          2.3 kilometer
      Water loss rate:                                        2.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 50.0 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               2.2 degC
      Flowrate per production well:                          63.0 kg/sec
      Injection well casing ID:                               7.000 in
      Production well casing ID:                              7.000 in
      Number of times redrilling:                             0
      Power plant type:                                       Single-Flash


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        500.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0680 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Multiple Parallel Fractures Model
      Bottom-hole temperature:                              179.60 degC
      Fracture model = Square
      Well separation: fracture height:                     900.00 meter
      Fracture area:                                     810000.00 m**2
      Reservoir volume:                              1000000000 m**3
      Reservoir hydrostatic pressure:                         22750.31 kPa
      Plant outlet pressure:                                100.00 kPa
      Injectivity Index:                                      5.00 kg/sec/bar
      Reservoir density:                                   2700.00 kg/m**3
      Reservoir thermal conductivity:                         2.70 W/m/K
      Reservoir heat capacity:                             1000.00 J/kg/K


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       177.6 degC
      Average Production Temperature:                       177.4 degC
      Minimum Production Temperature:                       176.1 degC
      Initial Production Temperature:                       176.1 degC
      Average Reservoir Heat Extraction:                     32.92 MW
      Production Wellbore Heat Transmission Model = Ramey Model
      Average Production Well Temperature Drop:               2.2 degC
      Average Injection Well Pump Pressure Drop:           1375.6 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      80.00 MUSD
         Drilling and completion costs per well:             40.00 MUSD
         Stimulation costs:                                   1.51 MUSD
         Surface power plant costs:                          13.37 MUSD
         Field gathering system costs:                        1.16 MUSD
         Total surface equipment costs:                      14.53 MUSD
         Exploration costs:                                  32.20 MUSD
      Total capital costs:                                  128.24 MUSD
      Annualized capital costs:                               6.41 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.88 MUSD/yr
         Power plant maintenance costs:                       0.41 MUSD/yr
         Water costs:                                         0.03 MUSD/yr
      Total operating and maintenance costs:                  1.32 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.13 MW/(kg/s)
      Maximum Total Electricity Generation:                   2.57 MW
      Average Total Electricity Generation:                   2.56 MW
      Minimum Total Electricity Generation:                   2.52 MW
      Initial Total Electricity Generation:                   2.52 MW
      Maximum Net Electricity Generation:                     2.46 MW
      Average Net Electricity Generation:                     2.45 MW
      Minimum Net Electricity Generation:                     2.41 MW
      Initial Net Electricity Generation:                     2.41 MW
      Average Annual Total Electricity Generation:           20.10 GWh
      Average Annual Net Electricity Generation:             19.24 GWh
      Initial pumping power/net installed power:              4.53 %
      Average Pumping Power:                                  0.11 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                176.14               0.1090            2.4091                7.3909
   2           1.0039                176.84               0.1090            2.4340                7.4263
   3           1.0051                177.04               0.1090            2.4412                7.4365
   4           1.0057                177.14               0.1090            2.4449                7.4417
   5           1.0061                177.21               0.1090            2.4474                7.4452
   6           1.0063                177.26               0.1090            2.4492                7.4478
   7           1.0066                177.30               0.1090            2.4506                7.4499
   8           1.0068                177.33               0.1090            2.4518                7.4515
   9           1.0069                177.36               0.1090            2.4528                7.4529
  10           1.0070                177.38               0.1090            2.4536                7.4541
  11           1.0072                177.40               0.1090            2.4544                7.4552
  12           1.0073                177.42               0.1090            2.4551                7.4561
  13           1.0074                177.44               0.1090            2.4557                7.4570
  14           1.0074                177.45               0.1090            2.4562                7.4578
  15           1.0075                177.47               0.1090            2.4567                7.4585
  16           1.0076                177.48               0.1090            2.4572                7.4591
  17           1.0077                177.49               0.1090            2.4576                7.4597
  18           1.0077                177.50               0.1090            2.4580                7.4603
  19           1.0078                177.51               0.1090            2.4583                7.4608
  20           1.0078                177.52               0.1090            2.4587                7.4613
  21           1.0079                177.53               0.1090            2.4590                7.4617
  22           1.0079                177.54               0.1090            2.4593                7.4621
  23           1.0080                177.55               0.1090            2.4596                7.4626
  24           1.0080                177.55               0.1090            2.4599                7.4629
  25           1.0081                177.56               0.1090            2.4601                7.4633
  26           1.0081                177.57               0.1090            2.4604                7.4637
  27           1.0081                177.58               0.1090            2.4606                7.4640
  28           1.0082                177.58               0.1090            2.4608                7.4643
  29           1.0082                177.59               0.1090            2.4611                7.4646
  30           1.0082                177.59               0.1090            2.4613                7.4649


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  19.1                       257.8                348.98                   0.27
   2                  19.2                       258.6                348.05                   0.53
   3                  19.3                       258.9                347.12                   0.80
   4                  19.3                       259.1                346.19                   1.06
   5                  19.3                       259.2                345.25                   1.33
   6                  19.3                       259.3                344.32                   1.60
   7                  19.3                       259.4                343.38                   1.86
   8                  19.3                       259.4                342.45                   2.13
   9                  19.3                       259.5                341.52                   2.40
  10                  19.3                       259.5                340.58                   2.67
  11                  19.4                       259.6                339.65                   2.93
  12                  19.4                       259.6                338.71                   3.20
  13                  19.4                       259.6                337.78                   3.47
  14                  19.4                       259.7                336.84                   3.73
  15                  19.4                       259.7                335.91                   4.00
  16                  19.4                       259.7                334.97                   4.27
  17                  19.4                       259.7                334.04                   4.54
  18                  19.4                       259.8                333.10                   4.80
  19                  19.4                       259.8                332.17                   5.07
  20                  19.4                       259.8                331.23                   5.34
  21                  19.4                       259.8                330.30                   5.60
  22                  19.4                       259.8                329.36                   5.87
  23                  19.4                       259.9                328.43                   6.14
  24                  19.4                       259.9                327.49                   6.41
  25                  19.4                       259.9                326.56                   6.67
  26                  19.4                       259.9                325.62                   6.94
  27                  19.4                       259.9                324.68                   7.21
  28                  19.4                       259.9                323.75                   7.48
  29                  19.4                       259.9                322.81                   7.74
  30                  16.2                       216.6                322.03                   7.97


                             ********************************
                             *  REVENUE & CASHFLOW PROFILE  *
                             ********************************
Year            Electricity             |            Heat                  |           Cooling                 |         Carbon                    |          Project
Since     Price   Ann. Rev.  Cumm. Rev. |   Price   Ann. Rev.   Cumm. Rev. |  Price   Ann. Rev.   Cumm. Rev.   |   Price   Ann. Rev.   Cumm. Rev.  | OPEX    Net Rev.      Net Cashflow
Start    (cents/kWh)(MUSD/yr) (MUSD)    |(cents/kWh) (MUSD/yr)    (MUSD)   |(cents/kWh) (MUSD/yr)    (MUSD)    |(USD/tonne) (MUSD/yr)    (MUSD)    |(MUSD/yr) (MUSD/yr)    (MUSD)
________________________________________________________________________________________________________________________________________________________________________________________
  1      0.00          -128.24   0.00     |    0.00     0.00         0.00    |    0.00     0.00         0.00     |    0.00     0.00         0.00     |  0.00     -128.24     -128.24
  2      5.50          -0.27   1.05     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.27     -128.51
  3      5.50          -0.27   2.11     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.27     -128.78
  4      5.50          -0.26   3.17     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -129.04
  5      5.50          -0.26   4.23     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -129.30
  6      5.50          -0.26   5.29     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -129.56
  7      5.50          -0.26   6.35     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -129.82
  8      5.50          -0.26   7.41     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -130.08
  9      5.50          -0.26   8.48     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -130.34
 10      5.50          -0.26   9.54     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -130.60
 11      5.50          -0.26  10.61     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -130.86
 12      5.50          -0.26  11.67     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -131.12
 13      5.50          -0.26  12.74     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -131.37
 14      5.50          -0.26  13.80     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -131.63
 15      5.50          -0.26  14.87     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -131.89
 16      5.50          -0.26  15.93     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -132.15
 17      5.50          -0.26  17.00     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -132.40
 18      5.50          -0.26  18.06     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -132.66
 19      5.50          -0.26  19.13     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -132.92
 20      5.50          -0.26  20.19     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -133.17
 21      5.50          -0.26  21.26     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -133.43
 22      5.50          -0.26  22.33     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -133.69
 23      5.50          -0.26  23.39     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -133.94
 24      5.50          -0.26  24.46     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -134.20
 25      5.50          -0.26  25.53     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -134.45
 26      5.50          -0.26  26.59     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -134.71
 27      5.50          -0.26  27.66     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -134.96
 28      5.50          -0.26  28.73     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -135.22
 29      5.50          -0.26  29.79     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -135.48
 30      5.50          -0.26  30.86     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.32     -0.26     -135.73
